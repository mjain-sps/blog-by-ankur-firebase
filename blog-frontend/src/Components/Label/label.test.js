import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import LabelComponent from "./label.components";

describe("Label component", () => {
  test("label component snapshot testing", () => {
    const dummyProps = { htmlFor: "input-id" };
    const { container } = render(<LabelComponent {...dummyProps} />);

    expect(container).toMatchSnapshot();
  });
});
