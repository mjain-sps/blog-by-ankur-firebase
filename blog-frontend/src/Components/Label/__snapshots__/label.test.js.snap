// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component label component snapshot testing 1`] = `
<div>
  <label
    class="sc-bdnxRM jVUQSi"
    for="input-id"
  />
</div>
`;
